ui/src/app/applications/components/application-resource-tree/application-resource-tree.tsx

```diff
@@ -514,97 +541,119 @@ function renderPodGroup(props: ApplicationResourceTreeProps, id: string, node: R
-                                                action: () => props.onNodeClick(pod.fullName)
-                                            },
-                                            {
-                                                title: (
-                                                    <React.Fragment>
-                                                        <i className='fa fa-align-left' /> Logs
-                                                    </React.Fragment>
-                                                ),
-                                                action: () => {
-                                                    props.appContext.apis.navigation.goto('.', {node: pod.fullName, tab: 'logs'}, {replace: true});
-                                                }
-                                            },
-                                            {
-                                                title: (
-                                                    <React.Fragment>
-                                                        <i className='fa fa-times-circle' /> Delete
-                                                    </React.Fragment>
-                                                ),
-                                                action: () => {
-                                                    deletePodAction(pod, props.appContext, props.app.metadata.name, props.app.metadata.namespace);
-                                                }
-                                            }
-                                        ]}
-                                    />
-                                ))}
-                            </div>
-                            <div className='application-resource-tree__node--lower-section__pod-group__label'>PODS</div>
+                {[podGroupHealthy, podGroupDegraded, podGroupInProgress].map((pods, index) => {
+                    return (
+                        <div key={index} className={`application-resource-tree__node--lower-section__pod-group`}>
+                            {renderPodGroupByStatus(props, node, pods, showPodGroupByStatus)}
                         </div>
-                    </div>
-                )}
+                    );
+                })}
             </div>
         </div>
     );
 }
 
+function renderPodGroupByStatus(props: ApplicationResourceTreeProps, node: any, pods: models.Pod[], showPodGroupByStatus: boolean) {
+    return (
+        <div className='application-resource-tree__node--lower-section__pod-group__pod-container__pods'>
+            {pods.length !== 0 && showPodGroupByStatus ? (
+                <React.Fragment>
+                    <div style={{cursor: 'none'}} className={`pod-view__node__pod pod-view__node__pod--${pods[0].health.toLowerCase()}`}>
+                        <PodHealthIcon state={{status: pods[0].health, message: pods[0].health}} key={pods[0].uid} />
```

Comment:
Could you explain the behaviour when you hover over the health icon?  The cursor disappeared.

Reply:
Fixed. Remove the ` style={{cursor: 'none'}} ` and added the tooltip on the icon and  text.